.github/workflows/http-plugin-py-cd.yaml

```
@@ -0,0 +1,59 @@
+name: http-plugin-py-cd
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+    paths:
+      - "implementations/py/**"
+
+jobs:
+  cd:
+    name: http-plugin-py-cd
+    if: |
+      github.event.pull_request.merged &&
+      startsWith(github.event.pull_request.title, '[CD]') != true &&
```

shouldn't this be:
```suggestion
      startsWith(github.event.pull_request.title, '[CD]') == true &&
```
also, just wondering: do we want to have the same pattern as other implementations? Meaning that when we add `[CD]` in the PR we do a release of *all* implementations, or we would like to do this 1:1 (1 PR, 1 Release)

I don't have strong opinions but thought its worth sharing this :smile: